Classes/Issues/Comments/Reactions/ReactionsMenuViewController.swift

```diff
@@ -69,22 +76,27 @@ final class ReactionsMenuViewController: UICollectionViewController,
         collectionView?.register(EmojiCell.self, forCellWithReuseIdentifier: reuseIdentifier)
         collectionView?.reloadData()
         collectionView?.layoutIfNeeded()
+        let reactionsInRow = sectionedReactions.first?.count ?? 0
```

Comment:
Can we at least throw an `assertionFailure` if `first` here is `nil`? It wouldn't make sense for this to have a `0` width in that case...

Reply:
Fixed 👍